tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookHelperTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Security.Authentication;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Bot.Schema;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Bot.Builder.Adapters.Facebook.Tests
+{
+    public class FacebookHelperTests
+    {
+        private const string AuthTokenString = "authToken";
+        private readonly Uri _validationUrlString = new Uri("http://contoso.com");
+
+        [Fact]
+        public void ActivityToFacebookShouldReturnMessageOptionsWithMediaUrl()
+        {
+            var uri = File.ReadAllText(Directory.GetCurrentDirectory() + @"\Files\Activities.json");
+            var activity = JsonConvert.DeserializeObject<Activity>(uri);
+
+            activity.Attachments = new List<Attachment> { new Attachment(contentUrl: "http://example.com") };
+            var messageOption = FacebookHelper.ActivityToFacebook(activity);
+
+            Assert.Equal(activity.Conversation.Id, messageOption.Recipient.Id);
+            Assert.Equal(activity.Text, messageOption.Message.Text);
+            Assert.Equal(new Uri(activity.Attachments[0].ContentUrl), messageOption.Message.Attachment.Payload.Url);
+        }
+
+        [Fact]
+        public void ActivityToFacebookShouldThrowErrorWithTwoOrMoreAttachments()
+        {
+            var activity = new Activity()
+            {
+                Conversation = new ConversationAccount()
+                {
+                    Id = "testId",
+                },
+                Attachments = new List<Attachment> { new Attachment(contentUrl: "http://example.com"), new Attachment(contentUrl: "http://example.com") },
+            };
+
+            Assert.Throws<Exception>(() => { var messageOptions = FacebookHelper.ActivityToFacebook(activity); });
```

Comment:
Here the variable can be removed as it isn't used.
```suggestion
            Assert.Throws<Exception>(() => { FacebookHelper.ActivityToFacebook(activity); });
```